mirage/factories/node.ts

```diff
@@ -186,6 +189,9 @@ export default Factory.extend<MirageNode & NodeTraits>({
     anonymized: trait<MirageNode>({
         _anonymized: true,
     }),
+    hasBoaEnabled: trait<MirageNode>({
+        boaEnabled: true,
+    }),
```

Comment:
Does this need to be a trait, or could we just set `boaEnabled` to true?

Reply:
Ahh, I was going to have this trait do a bit more (e.g. add a `.boa` file to the node's OsfStorage), but maybe having a trait do all of that is a bit much. Will remove